packages/constellation/src/components/Compositions/Table/Cells/BaseCell/BaseCell.types.ts

```diff
@@ -0,0 +1,11 @@
+import { FC, ReactNode } from 'react'
+
+type Props = {
+  isFooter?: boolean
+  isHeader?: boolean
+  postfix?: ReactNode
+  prefix?: ReactNode
```

Comment:
Could you add some descriptions for these? I'm not 100% clear on what the `postfix` and `prefix` props are and how they differ from passing content via `children` exclusively.